example-application/api.js

```diff
@@ -56,19 +56,17 @@ const defineRoutes = (expressApp) => {
       }
 
       // verify user existence by calling external Microservice
-      const existingUserResponse = await axios.get(
-        `http://localhost/user/${req.body.userId}`,
-        {
-          validateStatus: false,
+      try {
+        const user = await getUserFromUserService(req.body.userId);
+      } catch (error) {
+        const { response } = error;
```

Comment:
Don't you prefer Axios not to throw rather return status. Then you check if != 200 and throw AppError(... status) then our error handler just copy the status to the response. Just a thought

Reply:
I think I'm ok both ways... 
anyway, changed it to throw AppError

Reply:
Now I remember, `axiosRetry` requires `axios` to throw an error. 
`validateStatus: false` implicitly disable `axiosRetry`

Reply:
@mikicho axios-retry will retry also without errros, see the param "retryCondition":
https://www.npmjs.com/package/axios-retry

Reply:
I don't think so.
They implement it the "rejected" interceptor:
https://github.com/softonic/axios-retry/blob/master/es/index.js#L180

with `validateStatus: false` `axios` doesn't even call it.